### PR TITLE
POC

### PR DESCRIPTION
**Pull request by kkonuganti-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/acsp/manage/users/interceptor/AcspDataRetrievalPermissionInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Component
+public class AcspDataRetrievalPermissionInterceptor implements HandlerInterceptor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger( StaticPropertyUtil.APPLICATION_NAMESPACE );
+
+    @Override
+    public boolean preHandle( final HttpServletRequest request, final HttpServletResponse response, final Object handler ) {
+        if ( !isOAuth2Request() ){
```
[the remark this answers is not in the document]
Comment (krishna-patel-ch) [reply src/main/java/uk/gov/companieshouse/acsp/manage/users/interceptor/AcspDataRetrievalPermissionInterceptor.java]: Great spot. I've made the change

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by bmchugh-ch — APPROVED** [review]
Nice work.

**Pull request merged**